src/lib/actor.ts

```diff
@@ -53,6 +53,30 @@ export const getApifyStorageClient = async (
 	});
 };
 
+/**
+ * Returns instance of ApifyClient.
+ * @param options - ApifyClient options
+ */
+export const getApifyClient = async (options: ApifyClientOptions = {}): Promise<ApifyClient> => {
```

Comment:
Ideally we'd still use the `getLoggedInClientOrThrow` method with some adjustments for using process.env.APIFY_TOKEN var. I can help with that if needed, it is also an open issue right now